internal/balancergroup/balancergroup.go

```diff
@@ -165,6 +165,20 @@ func (sbc *subBalancerWrapper) stopBalancer() {
 	sbc.balancer = nil
 }
 
+func (sbc *subBalancerWrapper) gracefulSwitch(builder balancer.Builder) {
```

Comment:
Nit: move this above `stopBalancer`. I often treat methods `close`/`stop` as code boundaries.

Reply:
Done.